### ISSUE TITLE
Swagger generation needs Java 8 

### ISSUE THREAD

**Issue by krllstdn** [mild-blue/txmatching#1091]
During recent installation of Txmatching on my machine I noticed that openapi-generator needs Java 8, with newer versions it didn't work. In [openapi-generator README](https://github.com/OpenAPITools/openapi-generator) is also mentioned Java 8, but mostly for Macs. I had a couple of problems during installation so I am not totally sure that this is the case. Please check that

**Comment by kubantjan** [comment]
with older versions of JAVA than 8? that would not be a problem, nobody has older version than 8, important is whether it works with NEWER versions of JAVA?

**Comment by krllstdn** [comment]
@kubantjan I tried newer versions but it didn't want to work. 
Ou, I see, I wrote "older versions" in the comment above, sorry. Will be corrected.

**Comment by krllstdn** [comment]
openai mi vygeneroval takovou odpoved :D

OpenAPI Generator is a code generation tool that uses the OpenAPI Specification (OAS) to generate a variety of output types, including client libraries, server stubs, and documentation. The OpenAPI Generator project requires Java 8 because it uses features and APIs that are only available in Java 8 or earlier. For example, the project uses the Optional class, which was introduced in Java 8, and it uses older APIs for generating JSON and XML. If you try to use OpenAPI Generator with a newer version of Java, it may not work because these features and APIs may not be available.

**Comment by kubantjan** [comment]
have a look whether there is some other tool, that would not require java 8, maybe there is some other state of the art tool to generate the swagger from flask

**Comment by krllstdn** [comment]
@kubantjan yes, there are two tools that are able to generate swagger from flask 
- flask-swagger-generator
- flask_restplus

**Comment by kubantjan** [comment]
Lets use flask_restplus. It looks much more used. Please do it as part of this issue @krllstdn

**Comment by krllstdn** [comment]
I have tried following generators
- [swagger-codegen](https://github.com/swagger-api/swagger-codegen)
- [swagger-typescript-api ](https://github.com/acacode/swagger-typescript-api)
- [swagger-typescript-codegen](https://github.com/mtennoe/swagger-typescript-codegen)

none of them fully covers our needs(service suffixes, model suffixes, file naming convention, separate files(two of them generate everything to one file)) All listed above means signifikant changes in frontend, which we don't want.

I installed from the beginning openapi-generator with java-19-jre and, comically enough, it did run. 
Maybe during all those package manipulations some package was installed that enabled me to run openapi-generator now. IDK. But now it works.
